cdap-client/src/main/java/co/cask/cdap/client/ArtifactClient.java

```diff
@@ -97,7 +99,7 @@ public ArtifactClient(ClientConfig config) {
    * @throws UnauthenticatedException if the request is not authorized successfully in the gateway server
    * @throws NotFoundException if the namespace could not be found
    */
-  public List<ArtifactSummary> list(Id.Namespace namespace)
+  public List<ArtifactSummary> list(NamespaceId namespace)
```

Comment:
can't just change these method signatures, it will break integration tests. Need to add a corresponding method and deprecate this. I'm doing this already in a separate PR, you can remove the client changes.